colossalai/shardformer/layer/normalization.py

```diff
@@ -85,19 +234,27 @@ def from_native_module(module: nn.LayerNorm, *args, **kwargs) -> nn.Module:
 
         layernorm.weight = module.weight
         layernorm.bias = module.bias
-        return layernorm
 
+        return FusedLayerNorm(layernorm=layernorm)
```

Comment:
Apex's FusedRMSNorm is already a `torch.nn.Module`. Why wrap it?

Reply:
Weight and bias couldn't be hooked into Gemini's torch_function, making it unable to be properly converted to the 'hold afterbackward' state. I encapsulated it and added a view operation to both bias and weight.

Reply:
You can simply extend this class and override some affected methods like `forward()`